### DISCUSSION TITLE
djs/ws

### DISCUSSION THREAD

**Discussion by JuanPablo2655** [discordjs/discord.js#7681] [General]
I'm curious about the plan with the package and when the development is going to start. I'm assuming it's going into development before the rewrite of the main wrapper lib and saw a few planned features in https://github.com/discordjs/discord.js/pull/7204#issuecomment-1033865759.

Also thank to you guys, I'm in uni getting my cs degree!

**Comment by Jiralite** [comment]
It happened! https://discord.js.org/docs/packages/ws
